packages/spacecadetpinball-git/spacecadetpinball-git.pacscript

```diff
@@ -0,0 +1,82 @@
+#  __      __________   ______  
+# /  \    /  \_____  \ /  __  \ 
+# \   \/\/   //  ____/ >      < 
+#  \        //       \/   --   \
+#   \__/\  / \_______ \______  /
+#        \/          \/      \/ 
+maintainer="wizard-28 <wiz28@pm.me>"
+
+name="spacecadetpinball-git"
+pkgname="spacecadetpinball"
+pkgdir="/usr/src/pacstall/${name}"
+description='Reverse engineered port of "3D Pinball for Windows – Space Cadet" to Linux'
+url="https://github.com/k4zmu2a/${pkgname}.git"
+depends="timidity freepats libsdl2-2.0-0 libsdl2-mixer-2.0-0"
+build_depends="libsdl2-dev libsdl2-mixer-dev unrar cmake"
+breaks="${pkgname} ${pkgname}-bin ${pkgname}-app ${pkgname}-deb"
+
+pkgver() {
+  git ls-remote "${url}" master | cut -f1 
+}
+
+prepare() {
+  # Download game data file
+  wget -q 'https://archive.org/download/SpaceCadet_Plus95/Space_Cadet.rar'
+
```

Comment:
Add `pkgver`

Reply:
It's already there...

Reply:
The function yes, but where's the `version=$(pkgver)`?